CONTRIBUTING.md

```diff
@@ -31,12 +31,7 @@ We track our current work items using GitHub [project cards](https://github.com/
 
 ## Contribution options
 
-The OSCAL project is producing several types of deliverables, including the following:
-- *Schemas* for the OSCAL component models
-- *Schematron definitions*, which are basically an extension of the XML schemas that provide more validation capabilities
-- *XSL templates* for production of human-readable versions of OSCAL XML content
-- *CSS*, so people who are developing catalogs and profiles using XML tools can use CSS for data entry, which offers a much more usable interface
-- *Documentation* to define the OSCAL component models, capture the operational model of how to use OSCAL, and explain how you can convert existing content (catalogs, profiles, etc.) into OSCAL formats
+The OSCAL project is producing several types of deliverables, organized into several GitHub repositories as explained [in the project page](https://pages.nist.gov/OSCAL/about/projects/).
 
 Contributions are welcome in any of these areas. For information on the project's current needs and priorities, see the project's GitHub issue tracker (discussed below). Please refer to the [guide on how to contribute to open source](https://opensource.guide/how-to-contribute/) for general information on contributing to an open source project.
```

Comment:
The statement "Contributions are welcome in any of these areas" made perfect sense before 'the areas' were removed from the file. Maybe "Contributions are welcome in any of these repositories" or "Contributions to developing and maintaining these deliverables are welcome in the respective repositories"

Reply:
Will address in another commit coming soon.